pavelib/assets.py

```diff
@@ -292,6 +301,27 @@ def compile_templated_sass(systems, settings):
         print("\t\tFinished preprocessing {} assets.".format(system))
 
 
+def process_npm_assets():
+    """
+    Process vendor libraries installed via NPM.
+    """
+    # Skip processing of the libraries if this is just a dry run
+    if tasks.environment.dry_run:
+        return
+
+    # Ensure that the vendor directory exists
+    sh('mkdir -p {vendor_dir}'.format(
+        vendor_dir=STATIC_VENDOR_DIRECTORY,
+    ))
+
+    # Copy each file (if changed) to the vendor directory
+    for library in NPM_INSTALLED_LIBRARIES:
+        sh('cp -u node_modules/{library} {vendor_dir}'.format(
```

Comment:
Relying on timestamps to do copies seems like a bad idea.  There are lots of cases where the timestamps may not have updated correctly causing unexpected behavior.